doc/source/whatsnew/v1.0.0.rst

```diff
@@ -436,6 +436,7 @@ Groupby/resample/rolling
 - Bug in :meth:`DataFrame.groupby` not offering selection by column name when ``axis=1`` (:issue:`27614`)
 - Bug in :meth:`DataFrameGroupby.agg` not able to use lambda function with named aggregation (:issue:`27519`)
 - Bug in :meth:`DataFrame.groupby` losing column name information when grouping by a categorical column (:issue:`28787`)
+- Bug in :meth:`RollingGroupby.quantile` ignoring ``interpolation`` keyword argument (:issue:`28779`)
```

Comment:
i don’t think this will render, can u see how we reference rolling in other notes

Reply:
Sure, I'll find another example in the notes and imitate it.

Reply:
I changed the reference to `DataFrameGroupBy.rolling().quantile()`, similarly to the reference to `DataFrameGroupBy.rolling().apply()` here: https://pandas.pydata.org/pandas-docs/version/0.23.4/whatsnew.html#groupby-resample-rolling.  Hopefully that will work better?